package.json

```diff
@@ -62,6 +62,7 @@
         "@babel/preset-env": "^7.12.10",
         "@babel/preset-flow": "^7.12.1",
         "@trezor/blockchain-link": "^1.0.17",
+        "@trezor/connect-common": "^0.0.2",
```

Comment:
you need to add it to "extendedDependency" as well (to expose this dependency in npm-extended package)

Reply:
👍 added